### DISCUSSION TITLE
Extension to relational databases

### DISCUSSION THREAD

**Discussion by jakubpeleska** [pyg-team/pytorch-frame#273] [Q&A] [answered]
Hello, is there a plan to support primary/foreign key columns (as in [RelBench](https://github.com/snap-stanford/relbench/blob/0433616ee94003fb15a4ac4d633e499d0f129077/relbench/external/graph.py#L63)), or is this beyond the scope of this library?

**Comment by weihua916** [comment, marked as answer]
Thanks for your interest. For handling RelBench, we will combine PyTorch Frame and PyTorch Geometric. We are working on end-to-end example scripts. Stay tuned for the alpha release (early next year).
